components/01-basics/_collection-page.html

```diff
@@ -0,0 +1,54 @@
+<div class="slab slab--neutral" style="padding: 4rem">
+  <h1 class="u-padding--bottom">Collection page template</h1>
+  <p class="t-lead">Collection pages are Wagtail page templates. They contain an introduction, a sidebar with links to related or sibling content categories, and sections that list steps or procedures in the logical or chronological order that they apply. Each step links to its own page of instructions or information.</p>
+</div>
+
+<div class="slab" style="padding: 2rem">
+  <div class="container">
+
+    <div class="usa-width-one-third" style="float:right">
```

Comment:
There is a `.u-float-right` you can use here. (it's defined in fec/fec/static/scss/mixins/_utilities.scss)